scripts/addIngredientView.js

```diff
@@ -1,37 +1,97 @@
+var myBar = {
+  ingredients: [],
+  listTemplate: '',
+  localTemplate: '',
 
-console.log('loading view script');
-var possibleBarIngredients = {};
-
-possibleBarIngredients.name = [];
-possibleBarIngredients.id = [];
-
-possibleBarIngredients.fillInTypeahead = function (){
-  $.ajax({
-    type: 'GET',
-    url: 'http://www.cocktailbuilder.com/json/ingredientList?callback=ingredientList',
-    async: false,
-    jsonpCallback: 'ingredientList',
-    contentType: 'application/json',
-    dataType: 'jsonp',
-    success: function(data) {
-      console.log('in ajax success function');
-      possibleBarIngredients = data;
-      possibleBarIngredients.name = possibleBarIngredients.map(function(object){
-        return object.Name;
+  getTemplates: function () {
+    $.get('templates/ingredientsInMyBar_hbs.html')
+      .done( function(templateReturn){
+        myBar.listTemplate = Handlebars.compile(templateReturn);
       });
-      $('#autocomplete').autocomplete({
-        lookup: possibleBarIngredients.name,
-        minChars: 2,
-        onSelect: function () {
-          var newIngredient = '<li>' + $('#autocomplete').val() + '</li>';
-          $('#addedIngredients').append(newIngredient);
-          $('#autocomplete').val('');
-        }
+    $.get('templates/localBar_hbs.html')
+      .done( function(templateReturn){
+        myBar.localTemplate = Handlebars.compile(templateReturn);
+        myBar.ifLocalBarExists();
       });
-    },
-    error: function(e) {
-      console.log(e.message);
+  },
+
+  ifLocalBarExists: function () {
+    var storedBar = JSON.parse(localStorage.getItem('localBar'));
+    if(storedBar !== null && storedBar.length > 0){
+      myBar.ingredients = storedBar;
```

Comment:
one thing I don't like is that you're storing names of the ingredients in the local storage, not their ID's.